templates/rfc/classes.md

```diff
@@ -194,13 +194,15 @@ And a class consuming it.
 
 ```perl
 class Customer :isa(Person) :does(RoleStringify) {
-    slot $name  :param;              # must be passed to customer (:param)
-    slot $title :param = undef;      # optionally passed to constructor (:param, but with default)
+    slot $customer_id :param;
 
-    method name () {                # instance method
-        return defined $title ? "$title $name" : $name;
+    method name :overrides () {
+        my $name = $self->next::method;
+        $name .= " (#$customer_id)";
+        return $name;
     }
-    method to_string() { return $self->name }
+
+    method to_string($,$) { return $self->name }
```

Comment:
This is the only one I have an issue with. Why is there a `($,$)` in the signature? This method takes no arguments.